File: games/xu4/patches/patch-src_stats.cpp
$NetBSD: patch-src_stats.cpp,v 1.1.1.1 2012/01/08 15:52:12 wiz Exp $

Add missing header.

--- src/stats.cpp.orig	2005-08-24 05:55:35.000000000 +0000
+++ src/stats.cpp
@@ -18,6 +18,7 @@
 #include "spell.h"
 #include "tile.h"
 #include "weapon.h"
+#include <string.h>
 
 extern bool verbose;
 

File: sysutils/dusage/patches/patch-src_stats.rs
$NetBSD: patch-src_stats.rs,v 1.2 2026/08/30 08:57:44 he Exp $

Improve portability to 32-bit CPUs.
Reported upstream at https://github.com/mihaigalos/dusage/pull/67

--- src/stats.rs.orig	2026-08-30 08:17:54.285693493 +0000
+++ src/stats.rs
@@ -50,7 +50,7 @@ impl Stats {
             }
         };
 
-        let fragment_size: u64 = statvfs.fragment_size();
+        let fragment_size: u64 = statvfs.fragment_size().into();
 
         // Calculate disk usage stats
         let size_disk = blocks * fragment_size;
